src/api/Events.ts

```diff
@@ -274,4 +276,8 @@ export default class Events extends API {
       this.options({ method: "POST", body }).authorization({ sign: true })
     )
   }
+
+  async getCategories(): Promise<EventCategoryAttributes[]> {
+    return this.fetch(`/events_categories`)
```

Comment:
we are using REST as standard to name our endpoints please use `/events/categories` for this:

```suggestion
    return this.fetch(`/events/categories`)
```